curation_concerns-models/lib/curation_concerns/permissions.rb

```diff
@@ -0,0 +1,9 @@
+module CurationConcerns
+  module Permissions
```

Comment:
better: move `curation_concerns/file/permissions.rb` to `curation_concerns/permissions.rb` and move `curation_concerns/file/permissions` to `curation_concerns/permissions`.  I think I made a few changes in the modules that are already in `curation_concerns/file/permissions`